internal/bgp/stats.go

```diff
@@ -2,42 +2,60 @@ package bgp
 
 import "github.com/prometheus/client_golang/prometheus"
 
+type stat struct {
```

Comment:
Are the changes here just refactoring?  It'd be nice to split it into a separate commit to make it more clear what these changes were for.

Reply:
done, moved to a separate commit